src/test/java/com/techsophy/tsf/form/utils/TokenUtilsTest.java

```diff
@@ -146,6 +147,20 @@ void getPaginationResponsePayload()
         PaginationResponsePayload responsePayload = tokenUtils.getPaginationResponsePayload(page,list);
         assertThat(responsePayload).isNotNull();
     }
+    @Test
+    void getTenantTest()
+    {
+        Authentication authentication = Mockito.mock(Authentication.class);
```

Comment:
why these test cases are modified as part of this implementation

Reply:
in the process of converting from All args constructor to set method ,  code coverage is decreased sir.so i cover with this method